gomint-server/src/main/java/io/gomint/server/util/Bearing.java

```diff
@@ -84,7 +84,7 @@ public Bearing opposite() {
      */
     public static Bearing fromAngle( float angle ) {
         // Normalize angle
-        angle -= 90;
+        angle += 180;// Why -90?
```

Comment:
180 also is wrong. best would be to change north and south in the if statements below and getting rid of any further angle manipulation